nbbuild/antsrc/org/netbeans/nbbuild/ParseProjectXml.java

```diff
@@ -854,7 +854,9 @@ private void translateModuleAutoDeps(String myCNB, List<Dep> deps, List<URL> mod
             }
             File jar = entry.getJar();
             if (!jar.isFile()) {
-                log("Cannot translate according to " + moduleAutoDeps + " because could not find " + jar, Project.MSG_WARN);
+                if (!Boolean.valueOf(getProject().getProperty("nbbuild.nowarn.missing.autodeps"))) {
```

Comment:
Great attempt. I am also annoyed by these message. I am not sure whether introduction of a special `nbbuild.nowarn.missing.autodeps` property is a solution appreciated by everyone - e.g. nobody will notice this property. Yet everyone is annoyed by these messages when doing `ant clean` - would it be possible to set this property when doing `ant clean`?

Reply:
Yes, nobody will notice - that was the plan. If the property was visible, it would be too easy to turn off and miss important messages (which may point to bad dependencies).